
---
bool answer_received
int32 SID


